MODULE.bazel

```diff
@@ -2,34 +2,40 @@
 
 module(
     name = "rules_helm",
-    version = "0.3.2",
     compatibility_level = 1,
+    version = "0.3.2",
 )
 
 bazel_dep(name = "platforms", version = "0.0.7")
 bazel_dep(name = "bazel_skylib", version = "1.5.0")
-bazel_dep(name = "stardoc", version = "0.6.2", repo_name = "io_bazel_stardoc")
-bazel_dep(name = "rules_go", version = "0.45.1", repo_name = "io_bazel_rules_go")
+bazel_dep(name = "stardoc", repo_name = "io_bazel_stardoc", version = "0.6.2")
+bazel_dep(name = "rules_go", repo_name = "io_bazel_rules_go", version = "0.45.1")
 bazel_dep(name = "rules_oci", version = "1.7.2")
 
-# This is unfortunately required by `rules_oci`.
+# This is required by `rules_oci`.
```

Comment:
🥲

Reply:
This feels unfortunate to me since my rules have no reliance on `aspect-dev` rules but the design of either bzlmod or rules_oci are forcing me and all users to include this. I don't care if rules_oci uses it but why do I need to add it to this file?

Reply:
Yeah that's odd. @thesayyn does that sound like a bug to you?

Reply:
i am not what's the issue here; we already have aspect-bazel-lib in our dependencies. https://github.com/bazel-contrib/rules_oci/blob/21c48973f0e79744a617ba5f2014c5d5fca90f17/MODULE.bazel#L9

Reply:
Yeah I saw that too, but when I removed the dep from rules_helm's MODULE.bazel I saw the an `oci_pull` error claiming it couldn't find the dependency.

Reply:
Eh, sounds like a dependency leak, we use copy_to_directory in BUILD file stamped by oci_pull that must be it.

Reply:
Sounds like this is unintended? Is there an upstream issue I can link to in place of `unfortunately`? 😅

Reply:
https://github.com/bazel-contrib/rules_oci/issues/575